resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/SparkConfPropagateSuite.scala

```diff
@@ -42,7 +42,7 @@ private[spark] trait SparkConfPropagateSuite { k8sSuite: KubernetesSuite =>
       sparkAppConf.set("spark.kubernetes.executor.deleteOnTermination", "false")
 
       val log4jExpectedLog =
-        Seq("Reconfiguration complete for context", "at URI /opt/spark/conf/log4j2.properties")
+        Seq("Reconfiguration complete for context", "at URI /opt/spark/conf", "/log4j2.properties")
```

Comment:
We meet a kubernetes integration test failed when updating log4j2 from 2.17.1 to 2.17.2.

The failed message: `" did not contain "at URI /opt/spark/conf/log4j2.properties" The application did not complete, driver log did not contain str at URI /opt/spark/conf/log4j2.properties. (KubernetesSuite.scala:485)`

The failed reason:  this is the [patch](https://github.com/apache/logging-log4j2/commit/0f3510ab5f2371705b2f5cf52e9807fb5804052a#diff-ab4255c11b7414e8167314c787cb3ac7925e30bf212d5c1dd8bf06c93f9717aeR81) that changed the way to load log4j2.properties. After this patch, ConfigurationSource loads the actual log4j2.properties instead of the soft link. Thus we got `at URI /opt/spark/conf/..2022_03_17_04_21_55.822844723/log4j2.properties` in logs, not `at URI /opt/spark/conf/log4j2.properties`.